packages/compass-web/scripts/electron-proxy.js

```diff
@@ -159,17 +154,6 @@ class AtlasCloudAuthenticator {
     return new URL(url, 'http://localhost').pathname.startsWith('/v2/');
   }
 
-  async #getCSRFHeaders() {
```

Comment:
Drive-by, we moved this logic to the client some time ago and forgot to remove this